bundles/net.tourbook.common/src/net/tourbook/common/messages.properties

```diff
@@ -85,7 +85,7 @@ Color_Chooser_SelectedColor_Tooltip               = Selected color in the color
 
 ColumnFactory_Category_Quirky = Quirky
 
-ColumnModifyDialog_Button_Default2_Tooltip           = Set columns to it's defaults (selection, sorting, width, ...)
```

Comment:
I was not aware about the differences between it's vs its 😄 

https://www.google.com/search?q=its+vs+it%27s

Reply:
Yes, both have a different meaning